xla/stream_executor/rocm/hip_blas_lt.cc

```diff
@@ -176,8 +176,10 @@ absl::Status BlasLt::Init() {
   auto hip_compute_type = AsHipblasComputeType(compute_type);
   SE_HIPBLAS_RETURN_IF_ERROR(wrap::hipblasLtMatmulDescCreate(
       &hip_desc, hip_compute_type, hip_scale_type));
+
   // Wrap hipblas handle immediately, so it is cleaned up if an error occurs.
-  BlasLt::MatmulDesc desc(hip_desc, hip_compute_type, hip_scale_type);
+  BlasLt::MatmulDesc desc(hip_desc, hip_compute_type, hip_scale_type,
+        int32_t(epilogue) & int32_t(Epilogue::kBias));
```

Comment:
Please use static_cast<int32_t>() instead of int32_t()
Otherwise the linter complains and blocks merging.

Reply:
yes, I have fixed it